Merged the PR locally and I am getting the following error upon `rdpy` start:

```
rdpy    | Traceback (most recent call last):
rdpy    |   File "/usr/bin/rdpy-rdphoneypot.py", line 4, in <module>
rdpy    |     __import__('pkg_resources').run_script('rdpy==1.3.2', 'rdpy-rdphoneypot.py')
rdpy    |   File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 739, in run_script
rdpy    |     self.require(requires)[0].run_script(script_name, ns)
rdpy    |   File "/usr/lib/python2.7/site-packages/pkg_resources/__init__.py", line 1501, in run_script
rdpy    |     exec(script_code, namespace, namespace)
rdpy    |   File "/usr/lib/python2.7/site-packages/rdpy-1.3.2-py2.7-linux-x86_64.egg/EGG-INFO/scripts/rdpy-rdphoneypot.py", line 27, in <module>
rdpy    |     
rdpy    |   File "build/bdist.linux-x86_64/egg/rdpy/core/hpfeedslog.py", line 20, in <module>
rdpy    | ImportError: No module named hpfeeds
rdpy    | >>> 
rdpy exited with code 0
```

--- a/rdpy/core/hpfeedslog.py
+++ b/rdpy/core/hpfeedslog.py
@@ -37,7 +37,7 @@ class Level(object):
     
 _LOG_LEVEL = Level.DEBUG
 serverid=os.environ.get('SERVERID')
-
+hpc=None
 
 if (os.environ.get('HPFEEDS_SERVER') and os.environ.get('HPFEEDS_SECRET') and os.environ.get(
         'HPFEEDS_IDENT') and os.environ.get('HPFEEDS_PORT') and os.environ.get('HPFEEDS_CHANNEL') and os.environ.get('SERVERID')):

--- a/setup.py
+++ b/setup.py
@@ -43,5 +43,6 @@ setup(name='rdpy',
           	'qt4reactor',
           	'rsa',
           	'pyasn1',
+			'hpfeeds'
 	  	],
 )

